api/src/Entity/ContentNode/MaterialNode.php

```diff
@@ -94,6 +94,10 @@ public function copyFromPrototype($prototype, &$entityMap = null) {
             $materialItem = new MaterialItem();
             $this->addMaterialItem($materialItem);
 
+            /** @var MaterialList $materialList */
+            $materialList = $entityMap->get($itemPrototype->materialList);
+            $materialList->addMaterialItem($materialItem);
```

Comment:
Hat der Test ohne das nicht funtioniert?

Wir setzten die Materialliste jetzt 2x. Einmal hier und einmal in `MaterialItem->copyFromPrototype`.
Könnten wir nicht einfach im MaterialItem:
```
$materialList->addMaterialItem($this);
```

Reply:
Danke für den Input.
Ich konnte es nach MaterialItem.copyFromPrototype() verschieben - was ich auch besser finde.